.github/workflows/python-tests.yml

```
@@ -0,0 +1,34 @@
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      DB_USER: "test_user"
+      DB_PASSWORD: "test_password"
+      DB_NAME: "test_db"
+      REDIS_PASSWORD: "test_password"
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)